matchms/Scores.py

```diff
@@ -128,39 +128,46 @@ def calculate(self) -> Scores:
                                                            is_symmetric=self.is_symmetric)
         return self
 
-    def scores_by_reference(self, reference: ReferencesType) -> numpy.ndarray:
+    def scores_by_reference(self, reference: ReferencesType,
+                            sort: bool = False) -> numpy.ndarray:
         """Return all scores (not sorted) for the given reference spectrum.
 
         Parameters
         ----------
         reference
             Single reference Spectrum.
+        sort
+            Set to True to obtain the scores in a sorted way (relying on the
+            sorting function from the given similarity_function).
```

Comment:
Use correct name of function and link it
```suggestion
            :meth:`~.BaseSimilarity.sort` function from the given similarity_function).
```